utils/date-utils.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * @description The function checks the validity of the time.
+ * @param {string} time - time in format '15:00:00'.
+ * @returns {(boolean|Error)} returns true if the time format is correct, or returns an error if otherwise.
+ * @example
+ * // returns true
+ * timeValidator('14:00:00')
+ * // returns Error
+ * timeValidator('35:00:00')
+ **/
+export const timeValidator = (time) => {
+  const isValid = /^(2[0-3]|[01]?[0-9]):([0-5]?[0-9]):([0-5]?[0-9])$/.test(
+    time
+  );
+
+  if (isValid) {
+    return true;
+  } else {
+    throw new Error('Invalid time format');
+  }
+};
+
+/**
+ * @description The function returns the current day of the week.
+ * @returns {Number} returns the current day of the week according to local time, where 0 represents Sunday.
+ **/
+export const getCurrentWeekDay = () => {
+  return new Date().getDay();
+};
+
+/**
+ * @description The function set the time and parse the set date of the recurring event.
+ * @param {string} time - time of recurring event in format '15:00:00'.
+ * @returns {Number} returns the set date in milliseconds.
+ **/
+export const setTime = (time) => {
```

Comment:
avoid code duplication. More or less like this:
```
export const getTimeInMillisecond = (time) => {
  const splittedTime = time.split(':')

  const timeInMillisecond = new Date().setHours(...splittedTime)

  return timeInMillisecond
}
```

Reply:
can I pass parameters as strings to the setHours method?
I checked, it works) 
but still, is it right?